addons/hr_holidays/tests/test_out_of_office.py

```diff
@@ -47,7 +47,6 @@ def test_leave_ooo(self):
             'channel_partner_ids': [(4, partner.id), (4, partner2.id)],
             'public': 'private',
             'channel_type': 'chat',
-            'email_send': False,
```

Comment:
global remark on commit 3 ([REM] mail: remove "email_send" field on the mail channel): this is more than just removing "email send". We are removing the whole email notification support on channels, meaning they now work only as records notifying by chat until a global revamp with Discorduss. Only people that will be notified are people explicitly added as recipients (partner_ids field of mail.message).

All this information should be developed a bit and explained in commit message. Indeed this is important to state "what is the status of mail channel" after this commit as this is quite a strange model :) .